plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotConfig.java

```diff
@@ -251,6 +255,19 @@ public PinotConfig setTargetSegmentPageSize(DataSize targetSegmentPageSize)
         return this;
     }
 
+    @NotNull
+    public Map<String, String> getExtraHttpHeaders()
+    {
+        return extraHttpHeaders;
+    }
+
+    @Config("pinot.extra-http-headers")
+    public PinotConfig setExtraHttpHeaders(String headers)
+    {
+        extraHttpHeaders = ImmutableMap.copyOf(MAP_SPLITTER.split(headers));
```

Comment:
Needs some validation. e.g. key is not empty

Reply:
done.